tests/conftest.py

```diff
@@ -253,34 +253,37 @@ def test_assets_directory() -> pathlib.Path:
     return pathlib.Path(__file__).parent / "test_assets"
 
 
-@pytest.fixture(scope="package")
-def grpc_session_creation_kwargs():
+@pytest.fixture(scope="module")
```

Comment:
If init_kwargs has scope=session, then all fixtures it depends on (grpc_init_kwargs, grpc_channel) must have scope=session.

pytest prints errors when you get it wrong:

```
___________ ERROR at setup of test__channel__get_boolean_property__returns_default_value[grpc_init_kwargs] ____________
ScopeMismatch: You tried to access the module scoped fixture grpc_channel with a session scoped request object, involved factories:
tests\conftest.py:279:  def init_kwargs(request)
tests\conftest.py:263:  def grpc_init_kwargs(grpc_channel)
tests\conftest.py:256:  def grpc_channel()
```

Reply:
Thanks for the explanation, i missed to push this change, now i have updated the fixture to be scoped to `session`.